### PR TITLE
Add expansion resources section

### PR DESCRIPTION
**Pull request by bc-traciporter**
This section was removed from the POS Foundation Guide. We want to add it to the readme instead.
https://developer.bigcommerce.com/docs/495a2058c70b4-pos-foundation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,3 +101,11 @@ Looking to help the world's leading brands and the next generation of successful
 - [BigCommerce Dev Center](https://developer.bigcommerce.com/?source=pos-foundation) - Learn more about BigCommerce platform features, APIs and SDKs
 - [BigDesign](https://developer.bigcommerce.com/big-design/?source=pos-foundation) - BigCommerce's library of React components with sandbox tools
 - [Building BigCommerce Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro?source=pos-foundation) - Learn more about apps, from lightweight single-merchant customizations to fully-featured apps you can sell in the BigCommerce App Marketplace
+
+### Expansion resources
+* [Multi-Storefront Overview](https://support.bigcommerce.com/s/article/Multi-Storefront)
+* [Subscription Foundation README](https://github.com/bigcommerce/subscription-foundation/blob/main/README.md)
+* [Subscription Foundation Overview](/api-docs/partner/subscription-solutions/foundation-guide)
```
Comment (slsriehl) [inline README.md]:
```suggestion
* [Subscription Foundation Overview](https://developer.bigcommerce.com/api-docs/partner/subscription-solutions/foundation-guide)
```

File: README.md [hunk 2]
```diff
@@ -101,3 +101,11 @@ Looking to help the world's leading brands and the next generation of successful
...
+* [Channels API Reference](/api-reference/store-management/channels/channels/createchannel)
```
Comment (slsriehl) [inline README.md]:
```suggestion
* [Channels API Reference](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/createchannel)
```

File: README.md [hunk 3]
```diff
@@ -101,3 +101,11 @@ Looking to help the world's leading brands and the next generation of successful
...
+* [App API Accounts](/api-docs/getting-started/authentication/rest-api-authentication#app-api-accounts)
```
Comment (slsriehl) [inline README.md]:
```suggestion
* [App API Accounts](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#app-api-accounts)
```

File: README.md [hunk 4]
```diff
@@ -101,3 +101,11 @@ Looking to help the world's leading brands and the next generation of successful
...
+* [Guide to Building Apps](/api-docs/apps/guide/intro)
```
Comment (slsriehl) [inline README.md]:
```suggestion
* [Guide to Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
```

**Comment by slsriehl** [comment]
I added some suggestions @bc-traciporter

**Comment by becomevocal** [comment]
👍

**Pull request merged**